hw/bsp/dialog_da1469x-dk-pro/otp_tool.py

```diff
@@ -179,9 +180,9 @@ def otp_write_key(infile, index, segment, uart):
             else:
                 # read key from base64 encoded AES key file
                 buf = f.read()
-                if isBase64(buf):
-                    if len(buf) != 32:
-                        raise SystemExit("AES key file has incorrect length")
+                buf = base64.b64decode(buf)
```

Comment:
Now that isBase64() not used, could we remove it?

Reply:
yes

Reply:
else:
                # read key from base64 encoded AES key file
                buf = f.read()

                try:
                    buf = base64.decodestring(buf)
                except ValueError:
                    raise SystemExit("Invalid base 64 file")

                if len(buf) != 32:
                    raise SystemExit("AES key file has incorrect length")

Reply:
this would catch errors without spewing to stderr

Reply:
Done and new PS uploaded.